mirascope/openai/prompt.py

```diff
@@ -175,7 +174,9 @@ def stream(self) -> Generator[OpenAIChatCompletionChunk, None, None]:
         for chunk in stream:
             yield OpenAIChatCompletionChunk(chunk=chunk, tool_types=tools)
 
-    async def async_stream(self) -> AsyncGenerator[OpenAIChatCompletionChunk, None]:
+    async def async_stream(
+        self, **kwargs: Any
```

Comment:
update docstring to have "Args: ..." section with `**kwargs: Additional keyword arguments`.

Also do this for normal stream not just async stream (you've already added kwargs, just not the docstring)